packages/sdk/deploy/deploy.ts

```diff
@@ -313,25 +318,17 @@ const func: DeployFunction = async ({ run, ethers, getNamedAccounts, deployments
 
   tx = await fuseFeeDistributor._editCErc20DelegateWhitelist(
     [
-      constants.AddressZero,
```

Comment:
This is a consequence of the fixes to _setImplementationSafe?

Reply:
no, it's just the way it works - needs a pair of old and new implementation to whitelist the transition from one to the other